package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "local-home-virual-environment",
+  "version": "0.0.1",
+  "description": "",
+  "private": true,
+  "license": "Apache-2.0",
+  "scripts": {
+    "compile": "tsc",
+    "test": "ava --verbose",
+    "lint": "eslint ."
+  },
+  "dependencies": {
+    "@google/local-home-sdk": "^1.0.0",
+    "cbor": "^5.0.2",
+    "opc": "^1.1.3",
+    "xmldom": "^0.3.0",
+    "yargs": "^15.3.1"
+  },
+  "devDependencies": {
+    "@types/node": "^14.0.13",
+    "@types/xmldom": "^0.1.29",
```

Comment:
Same here, this is a dependency of the sample.

Reply:
Will remove

Reply:
Done